doc/source/user/basics.types.rst

```diff
@@ -12,6 +12,146 @@ Array types and conversions between types
 NumPy supports a much greater variety of numerical types than Python does.
 This section shows which are available, and how to modify an array's data-type.
 
+NumPy numerical types are instances of `numpy.dtype` (data-type) objects,
```

Comment:
`instance` has specific meaning in python: I would expect `type(np.int8)` now to be `np.dtype`, but this is not true - they all derive from `np.generic` I don't think it is helpful to suggest it is.

EDIT: darn, I see that this is simply all from the previous piece of text...